recipe/meta.yaml

```diff
@@ -7,16 +7,14 @@ package:
 source:
   fn: ncurses-{{ version }}.tar.gz
   url: http://ftp.gnu.org/pub/gnu/ncurses/ncurses-{{ version }}.tar.gz
-  md5: 8cb9c412e5f2d96bc6f459aa8c6282a1
+  md5: 98c889aaf8d23910d2b92d65be2e737a
```

Comment:
Probably should move away from `md5` if at all possible.

Reply:
Agreed, I'll try to check from now on